### PR TITLE
Updated the upload instructions in the docs.

### LINKED ISSUE #939: Update Submission Process in TraceBaseDocs
### Inspiration

The submission process has changed and been simplified. The [TraceBase application "Upload" page](https://tracebase.princeton.edu/DataRepo/upload) was updated (not yet deployed), but the [Markdown docs](https://princeton-lsi-researchcomputing.github.io/tracebase/Upload/How%20to%20Upload/) are still referencing the old process.

### Description

Update the markdown docs to reflect the new process.

### Alternatives

Since both pages are fairly similar in content, consider consolidating them into a single page. This would probably warrant some mock-ups of a new navigation menu item, submission/validation page, and docs.

### Comment

-----

### Assumptions

1. List of assumptions that the code will not explicitly address/check
2. E.g. We will assume input is correct (explaining why there is no validation)

### Limitations

1. A list of things this work will specifically not do
2. E.g. This feature will only handle the most frequent use case X

### Affected Components

- change: File path or DB table ...
- add: Environment variable or server setting
- delete: External executable or cron job

### Requirements

- [ ] 1. List of numbered conditions to be met for the feature
- [ ] 2. E.g. Every column/row must display a value, i.e. cannot be empty
- [ ] 3. Numbers for reference & checkboxes for progress tracking

#### Interface Change description

None provided

#### Code Change Description

None provided

#### Tests

- [ ] 1. A description of at least one test for each requirement above.
- [ ] 2. E.g. Test for req 2 that there's an exception when display value is ''
- [ ] 3. Numbers for reference & checkboxes for progress tracking

### PR DESCRIPTION
**Pull request by hepcat72**
## Summary Change Description

I also added a note to the nplcadmindocs (because it took me a long time to find where and how to deploy the documentation changes).

And I deployed the changes to github.

## Affected Issues/Pull Requests

- Resolves #939
- Merges into main

## Review Notes

See comments in-line.

## Checklist

This pull request will be merged once the following requirements are met.  The
author and/or reviewers should uncheck any unmet requirements:

- Review requirements
  - Minimum approvals: 1 
  - No changes requested
  - All blocking issues resolved by reviewers
  - Specific reviewers: @__add_username_here__
    
  - Review period: 2 days 
- Associated issue/pull request requirements:
  
  - [x] All requirements in affected issues marked "resolved" are satisfied
  - [x] All required pull requests are merged *(or none)*
- Basic requirements
  
  - [x] [All linters pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting)
  - [x] [All tests pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)
  - [x] All conflicts resolved
- Overhead requirements
  
  - [x] [New/changed method tests implemented/updated *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
  - [ ] [Updated *Unreleased* section of `changelog.md` *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/changelog.md)
  - [x] [Migrations created & committed *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)

### CODE REVIEW

File: TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md [hunk 1]
```diff
@@ -13,47 +13,27 @@ you initially submit for upload.
...
-3. Submit the [TraceBase Study Submission form](https://forms.gle/vEfJEfhPbCbkybpE7)
-   Complete the following:
-   1. Study Name
-   2. Link to Google Drive folder with study submission.
-   3. Mass Spectrometry Details (see [[How to label Mass Spec Run
-      Information]]).  Note, the LCMS-metadata file can be used to denote
-      multiple different operators, dates, methods, etc. The values entered into
-      the submission form are used as defaults for any samples that are not
-      included in the file.
+1. Download a submission template (a.k.a. a "Study doc")
```
Comment (lparsons) [inline TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md:16]: Perhaps "Create a submission template..." to convey they the template is customized?

File: TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md [hunk 2]
```diff
@@ -13,47 +13,27 @@ you initially submit for upload.
...
+    - Click the "Upload" tab at the top of any page of TraceBase.
+    - You can upload your peak annotation files (e.g. "AccuCor"), and the your
+      samples and compounds will be extracted and added to the template to give
+      you a head start.
```
Comment (lparsons) [inline TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md:20]: Actually, there is no way to get a "blank" template at the moment (Either an Animal/Sample Table or Peak Annotation file (e.g. Accucor or Isocorr) is required.). Perhaps change to something like:
"Upload your peak annotation files (e.g. "AccuCor") to generate a template. The
      samples and compounds will be extracted and added to the template to give
      you a head start."

File: TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md [hunk 3]
```diff
@@ -13,47 +13,27 @@ you initially submit for upload.
...
+
+2. Fill in the template with your data
```
Comment (lparsons) [inline TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md:22]: "Fill in the template with data about your samples"

File: TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md [hunk 4]
```diff
@@ -13,47 +13,27 @@ you initially submit for upload.
...
+    - Follow the directions under the "Fill In" tab to flesh out your study doc
+      with all the details of your study.
+
+3. Validate your Study doc
```
Comment (lparsons) [inline TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md:26]: "Check your "Study doc" for errors"

**Review by lparsons — APPROVED** [review]
Nice improvement and matches our current process. I made a few minor suggestions, take them or leave as you see fit.

File: TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md (hunk 1 above)
Comment (hepcat72) [reply TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md:16]:
```suggestion
1. Create a submission template (a.k.a. a "Study doc")
```

File: TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md (hunk 4 above)
Comment (hepcat72) [reply TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md:26]:
```suggestion
3. Check your Study doc for errors
```

File: TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md (hunk 2 above)
Comment (hepcat72) [reply TraceBaseDocs/TraceBaseDocs/Upload/How to Upload.md:20]:
```suggestion
    - Upload your peak annotation files (e.g. "AccuCor") to generate a template.
      The samples and compounds will be extracted and added to the template to
      give you a head start.
```

**Pull request merged**